frontend/packages/console-plugin-sdk/src/typings/dashboards.ts

```diff
@@ -76,15 +76,13 @@ namespace ExtensionProperties {
     popupTitle?: string;
   }
 
-  export interface DashboardsOverviewHealthCRDSubsystem extends DashboardsOverviewHealthSubsystem {
-    /** CRD resource which will be fetched and passed to healthHandler  */
-    resource: FirehoseResource;
+  export interface DashboardsOverviewHealthResourceSubsystem
```

Comment:
```suggestion
  export interface DashboardsOverviewHealthResourceSubsystem<R extends ResourcesObject>
```

`ResourcesObject` is defined in `k8s-watch-hook`